pkg/types/v1/config.go

```diff
@@ -209,6 +209,7 @@ type Partition struct {
 	Size            uint     `yaml:"size,omitempty" mapstructure:"size"`
 	FS              string   `yaml:"fs,omitempty" mapstrcuture:"fs"`
 	Flags           []string `yaml:"flags,omitempty" mapstrcuture:"flags"`
+	Label           string
```

Comment:
while this changes the struct, it should not affect anything, just stores the partitionLabel here which is info that we had but never used anywhere.